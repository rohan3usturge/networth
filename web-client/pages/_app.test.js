import App from "./_app";
import { shallow } from "enzyme";
import init from "../i18n";

jest.mock("../i18n");

describe("Footer", () => {
  it("should render properly and match snapshot", () => {
    const props = {
      Component: () => <span>Test</span>,
      pageProps: { locale: "en-ca" },
    };
    const wrapper = shallow(<App {...props} />);
    expect(wrapper).toMatchSnapshot();
    expect(init).toHaveBeenCalledWith("en-ca");
  });
});
