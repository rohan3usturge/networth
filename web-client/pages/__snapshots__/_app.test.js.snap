// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render properly and match snapshot 1`] = `
<Component
  locale="en-ca"
/>
`;
